test/e2e/ha/command_test.go

```diff
@@ -0,0 +1,552 @@
+package ha
+
+import (
+	"context"
+	"fmt"
+	"github.com/Juniper/contrail-operator/pkg/client/swift"
+	k8swait "k8s.io/apimachinery/pkg/util/wait"
+	"net/http"
+	"testing"
+	"time"
+
+	"github.com/operator-framework/operator-sdk/pkg/test"
+	"github.com/operator-framework/operator-sdk/pkg/test/e2eutil"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	core "k8s.io/api/core/v1"
+	meta "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	k8client "sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+
+	contrail "github.com/Juniper/contrail-operator/pkg/apis/contrail/v1alpha1"
+	"github.com/Juniper/contrail-operator/pkg/client/keystone"
+	"github.com/Juniper/contrail-operator/pkg/client/kubeproxy"
+	"github.com/Juniper/contrail-operator/test/logger"
+	"github.com/Juniper/contrail-operator/test/wait"
+)
+
+func TestHACommand(t *testing.T) {
+	//TODO: Unskip this test once Command works in HA mode
+	t.Skip()
+	ctx := test.NewContext(t)
+	f := test.Global
+	defer ctx.Cleanup()
+
+	namespace, err := ctx.GetOperatorNamespace()
+	require.NoError(t, err)
+
+	proxy, err := kubeproxy.New(f.KubeConfig)
+	require.NoError(t, err)
+
+	log := logger.New(t, namespace, f.Client)
+
+	if err := test.AddToFrameworkScheme(contrail.SchemeBuilder.AddToScheme, &contrail.ManagerList{}); err != nil {
+		t.Fatalf("Failed to add framework scheme: %v", err)
+	}
+
+	if err := test.AddToFrameworkScheme(core.AddToScheme, &core.PersistentVolumeList{}); err != nil {
+		t.Fatalf("Failed to add core framework scheme: %v", err)
+	}
+
+	if err := ctx.InitializeClusterResources(&test.CleanupOptions{TestContext: ctx, Timeout: cleanupTimeout, RetryInterval: cleanupRetryInterval}); err != nil {
+		t.Fatalf("Failed to initialize cluster resources: %v", err)
+	}
+
+	t.Run("given contrail operator is running", func(t *testing.T) {
+		err = e2eutil.WaitForOperatorDeployment(t, f.KubeClient, namespace, "contrail-operator", 1, retryInterval, waitForOperatorTimeout)
+		if err != nil {
+			log.DumpPods()
+		}
+		require.NoError(t, err)
+
+		w := wait.Wait{
+			Namespace:     namespace,
+			Timeout:       waitTimeout,
+			RetryInterval: retryInterval,
+			KubeClient:    f.KubeClient,
+			Logger:        log,
+		}
+
+		nodeLabelKey := "test-command-ha"
+		cluster := getHACommandCluster(namespace, nodeLabelKey)
```

Comment:
Please use unique storage path for test like
this https://github.com/Juniper/contrail-operator/blob/2b8f86038699426b893f5c3e0f02dfdd6e0c5af9/test/e2e/ha/core_contrail_services_test.go#L93

Reply:
It will ensure that tests are independent from each other

Reply:
Done